assets/features/CadastroProfissionaisdaSaude.feature

```diff
@@ -0,0 +1,39 @@
+#language:pt
+Funcionalidade: Cadastro de profissionais da saúde
+
+	Como Profissional da Saúde
+	Eu quero realizar cadastro dos meus dados pessoais/ profissionais.
+	Para que eu consiga realizar as operações no sistema.
+
+	Restrições:
+		1) O sistema deve estar acessivel.
+		2) Somente serão aceitos dados válidos.
+		3) Não permitir cadastros de pacientes com os mesmos dados.
+
+Esquema de Cenário: Cadastrando um profissional da saúde
+		Dado que eu esteja na tela de cadastro de profissionais da saúde
+		Quando eu inserir os meus dados (sendo válidos)
+		E preenchendo <nome>, <profissão>, <numero de conselho regional de saúde>, <email>, <telefone>
+		Então aparecerá uma mensagem me informando "<mensagem>"
```

Comment:
Se a mensgem sempre for a mesma para um dado válido, não é necessário colocar ela como exemplo.